cobalt/build/configs/media.gn

```diff
@@ -0,0 +1,5 @@
+# Build flag declared originally in build/config/features.gni
+# This flag is overridden to enable common video codecs playbacks.
+proprietary_codecs = true
+
+ffmpeg_branding = "Chrome"
```

Comment:
@borongc any suggestions as to why this is needed. Can you suggest a 1 line why this is needed?

Gets declared in chromecast/build/args/cast_shell_android.gn

Reply:
See https://source.chromium.org/chromium/chromium/src/+/main:android_webview/docs/aosp-system-integration.md;l=274?q=ffmpeg_branding%20f:%5C.md&ss=chromium%2Fchromium%2Fsrc

Reply:
Thanks for the links, added them to docs!